LifeProgress/Views/WelcomeScreen.swift

```diff
@@ -4,30 +4,51 @@ struct WelcomeScreen: View {
     @State private var showingProfile = false
 
     var body: some View {
-        VStack {
-            Image(systemName: "calendar")
-                .foregroundColor(.accentColor)
-                .font(.system(size: 100))
+        VStack(alignment: .leading) {
+            Spacer()
+            
+            Image("icon")
+                .resizable()
+                .frame(width: 100, height: 100)
+                .cornerRadius(12.0)
```

Comment:
> `12` is fine since Swift will just infer the type right?

Yep, Swift will implicitly refer the type.

> Also, can we rename the file to `OnboardingScreen.swift`?

Sure.